__tests__/app.test.js

```diff
@@ -247,4 +250,20 @@ describe('GET /api/articles/:article_id/comemnts', () => {
             })
         })
     });
+    test('200: should return an empty array of comments for a valid but non-existent article', () => {
+        return request(app)
+        .get("/api/articles/60/comments")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.articleComments).toEqual([])
+        })
+    });
+    test('404: should return "route not found" for incorrect path', () => {
```

Comment:
you don't need a new `404 - route not found` for each endpoint

your generic one on line #12 covers all cases